src/Features/Core/Portable/GenerateEqualsAndGetHashCodeFromMembers/GenerateEqualsAndGetHashCodeFromMembersCodeRefactoringProvider.cs

```diff
@@ -71,7 +71,7 @@ private async Task HandleNonSelectionAsync(CodeRefactoringContext context)
 
             // We offer the refactoring when the user is either on the header of a class/struct,
             // or if they're between any members of a class/struct and are on a blank line.
-            if (!syntaxFacts.IsOnTypeHeader(root, textSpan.Start) &&
+            if (!syntaxFacts.IsOnTypeHeader(root, textSpan.Start, out _) &&
```

Comment:
we could just have 2 overloads with and without out param?

Reply:
saying it since there is 1 usage case where we want out param and all other cases it is not needed so, it feels reversed that all has to be changed for 1 usage case.

Reply:
or create extension method in same namespace.

Reply:
The idea was to be consistent with other IsOnXXXHeader methods. And since having the out param is essentially free I feel like the clutter of having overload isn't worth it.

Reply:
Generally, being consistent doesn't cost us anything (since we can discard the result) and makes everything simpler to understand.